### PR TITLE
React loadable

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32804

**Comment by sabeurthabti** [comment]
can't believe I missed this

**Comment by mmahalwy** [comment]
@sabeurthabti you didn't! decided to merge it so we can see it on staging and get your thoughts

**Comment by sabeurthabti** [comment]
nah I meant this is exciting. Had a look and its awesome. 

shaved like 50kb off our main bundle. 

screen shot 2017-03-20 at 9 44 53 pm

**Comment by sabeurthabti** [comment]
Staging: https://www.webpagetest.org/result/170320_FE_1CXJ/1/performance_optimization/
Prod: https://www.webpagetest.org/result/170320_BW_1CYC/1/performance_optimization/

**Comment by mmahalwy** [comment]
Hmm, I think it's even better (the webpagetest) because staging is not using CDN but production does. It should even be better! 

The, the main bundle could be shaved more. Need to just find where there are things to shave. Any idea what would be best to dig into what webpack is adding?

File: src/components/TopOptions/index.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import Share from 'components/Share';
 import { surahType } from 'types';
 
-const styles = require('./style.scss');
-
 const TopOptions = ({ chapter }) => (
   <div className="row">
     <div className="col-md-4 hidden-xs hidden-sm">
-      <Title chapterNumber={chapter.id} className={styles.title} color={'#2CA4AB'} />
+      {/* NOTE: Caused about 7000 lines of code to accept all titles SVG */}
```
Comment (naveed-ahmad) [inline src/components/TopOptions/index.js]: does static assets ( svg ) count in LOC ?

Comment (mmahalwy) [inline src/components/TopOptions/index.js]: LOC?

Comment (naveed-ahmad) [inline src/components/TopOptions/index.js]: sorry, line of code :)

**Pull request merged**